salt/utils/parsers.py

```diff
@@ -1266,6 +1266,28 @@ def _mixin_setup(self):
         )
 
 
+class ExecutorsMixIn(six.with_metaclass(MixInMeta, object)):
+    _mixin_prio = 10
+
+    def _mixin_setup(self):
+        self.add_option(
+            '--module-executors',
```

Comment:
IMHO we need more documentation than this about what's happening here. I'm certain that I couldn't figure out what this does from the perspective of the average Salt user.

Reply:
This is already documented for the `salt` function, it would be the same thing.

https://docs.saltstack.com/en/latest/ref/executors/index.html

I can add an example with salt-call --module-executors in here?